vault/resource_kmip_secret_scope.go

```diff
@@ -67,15 +70,19 @@ func kmipSecretScopeCreate(d *schema.ResourceData, meta interface{}) error {
 
 func kmipSecretScopeRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*api.Client)
-	path := d.Get("path").(string)
-	expectedScope := d.Get("scope").(string)
+	path := d.Get("path").(string) + "/scope"
```

Comment:
nit: you may want to consider factoring computing of the scope root to a function since it is repeated in multiple places.

Reply:
Added utils `getScopePath` and `getRolePath` in 8da0ead